rpc/src/v1/traits/chain.rs

```diff
@@ -137,7 +137,7 @@ build_rpc_trait! {
 
         /// Gets transactions in the current mem pool.
         # [rpc(name = "chain_getPendingTransactions")]
-        fn get_pending_transactions(&self) -> Result<Vec<Transaction>>;
+        fn get_pending_transactions(&self, Option<u64>, Option<u64>) -> Result<Vec<Transaction>>;
```

Comment:
This RPC should return the inserted time of the last inserted transaction.
So the interface should be `fn get_pending_transactions(&self, Option<u64>, Option<u64>) -> Result<(Vec<Transaction>, u64)>>;

Reply:
I think declaring a new structure as below will be good.
```rust
PendingTransactions { transactions: Vec<Transaction>, last_timestamp: u64 }
```